package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-sdk",
-  "version": "5.0.1",
+  "version": "5.0.2",
```

Comment:
Yes, but as I stated last week, I plan to merge this PR directly on `master` once it is merged on `5.x`, and to rewrite the tag to make the `5.x` version of this SDK start at 5.0.2 instead of 5.0.0.
So I won't use the release tool and I want to merge this PR as-is, without modifying it before reaching master.

The idea is to avoid major versions jumps, as the `5.x` version has to be released before Kuzzle RC10. It is currently unusable anyway, until RC10 is released.